libraries/postgres/src/Database.js

```diff
@@ -387,15 +387,20 @@ class Database {
 
   async _checkDbSettings() {
     await this._withClient('admin', async client => {
-      const res = await client.query(`
-        SELECT datcollate AS collation
-        FROM pg_database 
-        WHERE datname = current_database()`);
-      const collation = res.rows[0].collation;
-      if (collation !== 'en_US.utf8' && collation !== 'en_US.UTF8') {
```

Comment:
That's definitely a weird behavior. Would it make more sense to just do `collation.toLowerCase()` and then only compare it with  `en_us.utf8` and `en_us.utf-8`? That seems a bit less complicated (i.e., creating an array and running an additional query) and more explicit.

Reply:
My first attempt at this stripped punctuation and lower-cased, and compared to `enusutf8` (hence the branch name), but that is just a guess at what is and is not acceptable.  I think the updated patch is a bit more precise about what we want: that the collation order the things we care about in the order we expect.  Which I think actually just means "not C" -- `sp_ES.utf16` would probably be fine.